From: Alexander Kulak <sa-dev@rainbow.by>
Date: Tue, 10 Apr 2018 16:15:32 +0300
Subject: system-wide blocklist

---
 ephy-uri-tester.c | 2 +-
 README.md | 4 ++--
 1 file changed, 3 insertion(+), 3 deletion(-)

diff --git a/ephy-uri-tester.c b/ephy-uri-tester.c
index 639b3aa..e16e23d 100644
--- a/ephy-uri-tester.c
+++ b/ephy-uri-tester.c
@@ -987,7 +987,7 @@ static gpointer inittcb(gpointer data)
 static void initex(WebKitWebExtension *ex, WebKitWebPage *wp)
 {
 	gchar *filter_path = g_build_filename(
-			g_get_user_config_dir(), APPNAME, "easylist.txt", NULL);
+			"/var/lib", APPNAME, "blocklist.txt", NULL);
 
 	if (g_file_test(filter_path, G_FILE_TEST_EXISTS))
 	{

diff --git a/README.md b/README.md
index ca088ae..e041607 100644
--- a/README.md
+++ b/README.md
@@ -10,9 +10,9 @@ most of code of this are from https://github.com/GNOME/epiphany/tree/master/embe
 	sudo make install
 
 then
-copy **easylist.txt** to ~/.config/wyebadblock/
+copy **blocklist.txt** to /var/lib/wyebadblock/
 
-wyebadblock only checks 'easylist.txt'
+wyebadblock only checks 'blocklist.txt'
 
 
 You can check if it works on http://simple-adblock.com/faq/testing-your-adblocker/
